contracts/OwnerOrManager.sol

```diff
@@ -1,72 +1,112 @@
+     * @param newManager new `_manager` address.
      */
     event ManagerTransferred(address indexed newManager);
 
     /**
-     * @notice Gets current _manager.
-     * @dev Returns current _manager address.1
-     * @return Current _manager address.
+     * @notice Gets current `_manager`.
+     * @return Current `_manager` address.
      */
     function manager() public view virtual returns (address) {
         return _manager;
     }
 
+    /**
+     * @notice Gets current `_pendingOwner`.
+     * @return Current `_pendingOwner` address.
+     */
+    function pendingOwner() external view virtual returns (address) {
+        return _pendingOwner;
+    }
+
+    /**
+    * @dev Throws if called by any account other than the `pendingOwner`.
+    */
+    modifier onlyPendingOwner() {
+        require(_msgSender() == _pendingOwner, "OwnerOrManager/caller-not-pendingOwner");
+        _;
+    }
+
     /**
      * @dev Throws if called by any account other than the manager.
      */
     modifier onlyManagerOrOwner() {
-        require(manager() == _msgSender() || owner() == _msgSender(), "Manager/caller-not-manager-or-owner");
+        require(manager() == _msgSender() || owner() == _msgSender(), "OwnerOrManager/caller-not-manager-or-owner");
         _;
     }
 
+    /**
+    * @dev Allows current owner to set the pendingOwner address.
+    * @param _newOwner Address to transfer ownership to.
+    */
+    function transferOwnership(address _newOwner) public override onlyOwner {
+        require(_newOwner != address(0), "OwnerOrManager/pendingOwner-not-zero-address");
+        _pendingOwner = _newOwner;
+    }
+
+    /**
+    * @dev Allows the pendingOwner address to finalize the transfer.
+    */
+    function claimOwnership() external onlyPendingOwner {
+        _setOwner(_pendingOwner);
+        _pendingOwner = address(0);
+    }
+
     /**
      * @notice Set or change of manager.
      * @dev Throws if called by any account other than the owner.
      * @param _newManager New _manager address.
      * @return Boolean to indicate if the operation was successful or not.
      */
-    function setManager(address _newManager) public onlyOwner returns (bool) {
-        _setManager(_newManager);
+    function setManager(address _newManager) external onlyOwner returns (bool) {
+        return _setManager(_newManager);
     }
 
     /**
      * @notice Set or change of manager.
-     * @dev Throws if called by any account other than the owner.
      * @param _newManager New _manager address.
      * @return Boolean to indicate if the operation was successful or not.
      */
     function _setManager(address _newManager) internal returns (bool) {
         address _previousManager = _manager;
-        require(_newManager != address(0), "Manager/manager-not-zero-address");
-        require(_newManager != _previousManager, "Manager/existing-manager-address");
+
+        require(_newManager != address(0), "OwnerOrManager/manager-not-zero-address");
```

Comment:
They should be able to clear the manager, I think setting to address(0) should be allowed

Reply:
Done: https://github.com/pooltogether/owner-manager-contracts/pull/2/commits/3a12659166cf1b2bf688be81d760cb1a76c6a045